website/src/controllers/user.js

```diff
@@ -300,10 +301,26 @@ api['delete'] = function(req, res, next) {
   var plan = res.locals.user.purchased.plan;
   if (plan && plan.customerId && !plan.dateTerminated)
     return res.json(400,{err:"You have an active subscription, cancel your plan before deleting your account."});
-  res.locals.user.remove(function(err){
+
+  function callback() {
```

Comment:
Kudos for being DRY and using a callback function, but can we name it something better? It's basically a wrapper for the remove user function, so maybe `removeUser()`?